README.md

```diff
@@ -1,32 +1,67 @@
-﻿# LoTAS Mod
-### Low Optimization Tool Assisted Speedrun Mod
+﻿# This project is on hiatus!
```

Comment:
can we please use words that everyone can understand...

Reply:
Hiatus is relatively common... I was also thinking about "hibernation" but thats also ehh... Not maintained sounds too harsh? idk

Reply:
`This project is no longer actively being developed!`